src/components/Visualizations/Graph/TrendGraph.jsx

```diff
@@ -72,7 +72,18 @@ const TooltipStylesDiv = styled.div`
     font-size: ${constants.fontSizeCommon};
   }
 `;
-
+const getHeroImg = (data) => {
+  try {
+    return (<img
+      className="heroImg"
+      src={`${process.env.REACT_APP_API_HOST}${heroes[data.hero_id]
```

Comment:
Probably the cause of this is just not having data.hero_id defined, so we should just set a default hero object so we don't try to read .img of undefined

Reply:
found the issue is actually due to payload[0], where payload is not defined, so I just added an additional check for that